docs/running-on-kubernetes.md

```diff
@@ -433,8 +433,6 @@ the cluster.
 When there exists a log collection system, you can expose it at Spark Driver `Executors` tab UI. For example,
 
 ```
-spark.executorEnv.SPARK_EXECUTOR_ATTRIBUTE_APP_ID='$(SPARK_APPLICATION_ID)'
-spark.executorEnv.SPARK_EXECUTOR_ATTRIBUTE_EXECUTOR_ID='$(SPARK_EXECUTOR_ID)'
```

Comment:
We don't need this from Apache Spark 4.0.0-preview2.

Reply:
Would be nice to still mention the feature of `spark.executorEnv.SPARK_EXECUTOR_ATTRIBUTE_*` here of using any Kubernetes environment variable.

Reply:
It's irrelevant to this PR because this PR specifically targets `Add SPARK_EXECUTOR_ATTRIBUTE_(APP|EXECUTOR)_ID if CUSTOM_EXECUTOR_LOG_URL is defined`.

However, of course, you can make a documentation contribution if you want, @EnricoMi . We will review it separately.

Reply:
Added in #47801